macOS compile: error: unknown type name 'CDirectLPT'

Hi!
It seems some recent changes again broke the macOS build. I had a quick look at it but don't understand really what might have broken it. The parport code seems to be old enough that it was around the last time I successfully built it. Perhaps some of the autoconf changes?

    ./configure --enable-debug=heavy --enable-core-inline --enable-avcodec
    make install

...snip...

```
Making install in parport
depbase=`echo parport.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	g++ -DHAVE_CONFIG_H -I. -I../../..  -I../../../include -Wno-strict-aliasing   -I/usr/local/include -I/usr/local/include/SDL -D_GNU_SOURCE=1 -D_THREAD_SAFE -I/usr/local/Cellar/ffmpeg/3.4.1/include  -g -O2  -std=gnu++11  -mmmx -msse -msse2  -Wno-strict-aliasing   -MT parport.o -MD -MP -MF $depbase.Tpo -c -o parport.o parport.cpp &&\
	mv -f $depbase.Tpo $depbase.Po
parport.cpp:336:5: error: unknown type name 'CDirectLPT'
                                CDirectLPT* cdlpt= new CDirectLPT(i, defaultirq[i],&cmd);
                                ^
parport.cpp:336:28: error: unknown type name 'CDirectLPT'
                                CDirectLPT* cdlpt= new CDirectLPT(i, defaultirq[i],&cmd);
                                                       ^
```

--- a/configure.ac
+++ b/configure.ac
@@ -257,6 +257,8 @@ case "$host" in
        if test x$have_sdl_net_lib = xyes -a x$have_sdl_net_h = xyes ; then
          LIBS="$LIBS -lws2_32"
        fi
+       dnl FEATURE: Whether to support direct parallel port passthrough
+       AC_DEFINE(C_DIRECTLPT, 1, [ Define to 1 if you want parallel passthrough support (Win32, Linux).])
        ;;
     *-*-darwin*)
        dnl We have a problem here: both Mac OS X and Darwin report 
@@ -269,6 +271,8 @@ case "$host" in
     *-*-linux*)
        AC_DEFINE(LINUX, 1, [Compiling on GNU/Linux])
        CXXFLAGS="$CXXFLAGS -D_XOPEN_SOURCE=700 -D_POSIX_C_SOURCE=200809L"
+       dnl FEATURE: Whether to support direct parallel port passthrough
+       AC_DEFINE(C_DIRECTLPT, 1, [ Define to 1 if you want parallel passthrough support (Win32, Linux).])
        ;;
     *-*-freebsd* | *-*-dragonfly* | *-*-netbsd* | *-*-openbsd*)
        dnl Disabled directserial for now. It doesn't do anything without
@@ -513,9 +517,6 @@ fi
 
 dnl FEATURE: Whether to support direct serial port passthrough
 AC_DEFINE(C_DIRECTSERIAL, 1, [ Define to 1 if you want serial passthrough support (Win32, Posix and OS/2).])
-
-dnl FEATURE: Whether to support direct parallel port passthrough
-AC_DEFINE(C_DIRECTLPT, 1, [ Define to 1 if you want parallel passthrough support (Win32, Linux).])
 
 dnl FEATURE: Whether to support SDL net, and emulate modem and IPX connections
 AH_TEMPLATE(C_SDL_NET,[Indicate whether SDL_net is present])

